### ISSUE TITLE
Council Request: Derry and Strabane District Council

### ISSUE THREAD

**Issue by sjta3b** [robbrad/UKBinCollectionData#862]
### Name of Council

Derry and Strabane District Council

### Example Address/Postcode

BT47 6BE

### Verification

- [X] I searched for similar issues at https://github.com/robbrad/UKBinCollectionData/issues?q=is:issue and found no duplicates
- [X] I have provided a tested working address/postcode/UPRN with bin collections available
- [X] I understand that this project is run by volunteer contributors and completion depends on numerous factors - even with a request, we cannot guarantee if/when your council will get a script

**Comment by robbrad** [comment]
@sjta3b looking at https://recycling.derrystrabane.com/services/bin-collections/#bin-collection - how do you work out which your day of collection is ?

**Comment by sjta3b** [comment]
Yeah, I've realised after looking at pretty much every council in the UK that mine is useless and can't be incorporated because they don't actually have a proper online system for finding out. The bin collection rotates on a weekly basis, but the only way to find out what actual day it's collected is to look outside and see when your neighbours put their bins out! Hopefully we'll reach the 21st century eventually. Thanks for trying to help.

**Comment by robbrad** [comment]
@sjta3b : can you make use of https://github.com/robbrad/UKBinCollectionData/wiki/Councils#google-calendar-public ?

If you make a Public google calander for this then use the link to the ICS file it could work

**Comment by robbrad** [comment]
Wont impliment due to PDFs - parsing PDFs is very unreliable - use Google calander approach instead
